package.json

```diff
@@ -48,13 +50,11 @@
     "node-sass": "^4.9.2",
     "openseadragon": "^2.4.0",
     "prop-types": "^15.6.2",
-    "react": "^16.8.3",
-    "react-dom": "^16.8.3",
     "react-full-screen": "^0.2.4",
     "react-i18next": "^10.2.0",
     "react-mosaic-component": "^2.1.0",
     "react-placeholder": "^3.0.1",
-    "react-redux": "^6.0.0",
```

Comment:
Can you say more about why we're downgrading and pinning `react-redux`?

Reply:
Yikes, thanks for catching that. I've removed this change to move back to what it was previously.